packages/backend/src/controllers/savedChartController.ts

```diff
@@ -0,0 +1,46 @@
+import { ApiErrorPayload } from '@lightdash/common';
+import { Get, Post } from '@tsoa/runtime';
+import express from 'express';
+import {
+    Controller,
+    Middlewares,
+    OperationId,
+    Path,
+    Request,
+    Response,
+    Route,
+    SuccessResponse,
+} from 'tsoa';
+import { projectService } from '../services/services';
+import { allowApiKeyAuthentication, isAuthenticated } from './authentication';
+import { ApiRunQueryResponse } from './runQueryController';
+
+@Route('/api/v1/projects/{projectUuid}/saved/{chartUuid}')
```

Comment:
Is this nested under projects because you need the `projectUuid` in `runViewChartQuery`? 

This is available on the `SavedChart` type returned from the saved chart model. 

By removing the projectUuid from the endpoint, it's easier for the frontend to make these API calls with less context.

Reply:
Done